markdown/articles/2020/12/20/historyofdoomsday.md

```diff
@@ -0,0 +1,1030 @@
+decks that would start to evolve into more Modern storm lists and Emidln was in the middle of them.
+
+Spanish Inquistion is a deck that a lot of people still hold very dear to their heart to this day. The idea of 
+the deck is to utilise fast mana and ritual effects combined with 'draw 4's' like {{Cruel Bargain)) and
+{{Infernal Contract}}. {{Land Grant}}, along with {{Culling the Weak}} and creatures like {{Phyrexian Walker}}
+were used to cheat on mana and then the deck finished off opponents with either Tendrils of Agony or
+{{Goblin Charbelcher}}. There was a point in time where the SI decks tried a number of cards in them to 
+increase the *business* count available as the draw 4's could be unreliable. Cards like {{Diablic Intent}} worked
+alongside the 'tallmen' approach with {{Shield Sphere}} and friends however this did stretch reliance on the creature
+aspect. Doomsday was explored by Emidln and BZK friends; Evenpence and Desolutionist as a way to incorporate additional 
+business into the shell to some success but the deck, although fast, did not achieve any top tier status.
+
+<row variant="pile">{{!Cruel Bargain}} {{!LP}} {{!DR}} {{!DR}} {{!ToA}}</row>
+
+IN 2006 Iggy pop got a large boost in the form of {{Leyline of the Void}} from Guildpact which allowed it to mitigate the
+balanced nature of Ill-Gotten Gains by removing the opposing hand from the game entirely. It could also just be
+used to make an opponent go hellbent from turn 1. A very key card was printed in Dissession in the form of 
+{{Infernal Tutor}}. This card allowed for a combo with Lion's Eye Diamond to achiveve hellbent status and 
+effectively become a {{Demonic Tutor}} in doing so. It also gave rise to other branch decks like Nausea which ran
+{{Spoils of the Vault}}, {{Chromatic Sphere}} and Helm of Awakening. The deck actually ran a single copy of
+Doomsday as a wish target with {{Meditate}} being the draw 4 of choice.
+
+
+## Legacy Evolves, Storm starts to diverge
+
+2006 was a pretty crucial year for storm with the introduction of the set Time Spiral. This was the first set since
+Scourge to revisit the Storm mechanic giving two key cards: {{Grapeshot}} and, more importantly, {{Empty the Warrens}}.
+These cards gave a huge boost to the decks trying to use them with the Goblins plan allowing players to go off much
+earlier with a quick Empty for 5 or so presenting 10+power on the board from turn 1. This also gave a huge boost vs the
+Control decks of the time which didn't have mana efficient board sweepers to deal with such aggression. Because of this
+new printing the BZK group started working on two key builds to make use of them. The first was developed by
+user Wastedlife (now better known as a certain Bryant Cook) who came up with this deck concept known as 
+'The Epic Storm' or TES. This started utilising principles such as including main deck protection and a diverse
```

Comment:
Epic should be stylized 'EPIC' as it was named for Bryant's legacy team, The EPIC Syndicate. 

I don't want to make it seem like I invented TES. I was around, I talked to Bryant a lot and did a ton of testing, but Bryant put in the most work on this. He was a brief member of BZK, and part of the reason we started Storm Boards (or "Team Storm" as we originally called it) was we were unhappy with The Source's quality of discussion and we wanted a group which would focus on evolving storm decks. It ceased being a team fairly early on when we opted for more inclusion.